settings/provisioner/foreman/foreman.spec

```diff
@@ -0,0 +1,68 @@
+---
+subparsers:
+    foreman:
+        help: Provision systems using 'foreman'
+        groups:
+            - title: foreman
+              options:
+                  foreman-user:
```

Comment:
since this is a FOREMAN provisioner there is no sense in putting "foreman-<flag>" for every flag we have...
This spec will cause a user to write:
ir-provision foreman --foreman-user
instead of: 
ir-provision foreman --user 
Is there a specific reason you added it like this?